packages/cli/e2e/__tests__/fixtures/trigger-project/checks.check.ts

```diff
@@ -0,0 +1,51 @@
+import { ApiCheck, CheckGroup } from 'checkly/constructs'
+
+const executionId = process.env.EXECUTION_ID!
+
+const prodBackendGroup = new CheckGroup('prod-backend-group', {
+  name: 'Production Backend Group (Trigger Test)',
+  locations: ['eu-central-1'],
+  tags: ['production', 'backend', executionId, 'additional-tag'],
+})
+
+new ApiCheck('prod-backend-check', {
+  name: 'Prod Backend Check (Trigger Test)',
+  activated: false,
+  request: {
+    url: 'https://www.google.com',
+    method: 'GET',
+    followRedirects: false,
+    skipSSL: false,
+    assertions: [],
+  },
+  group: prodBackendGroup,
+})
+
+new ApiCheck('prod-frontend-check', {
+  name: 'Prod Frontend Check (Trigger Test)',
+  activated: false,
+  request: {
+    url: 'https://www.google.com',
+    method: 'GET',
+    followRedirects: false,
+    skipSSL: false,
+    assertions: [],
+  },
+  tags: ['production', 'frontend', executionId, 'another-tag'],
+  // Log the secret environment variable passed to `checkly trigger`
+  localSetupScript: "console.log(process.env.SECRET_ENV)",
```

Comment:
The `localSetupScript` is marked as `deprecated`. Maybe, you can use the new `setupScript`

Reply:
Good idea, thanks! I updated the PR.